src/main/fc/tasks.c

```diff
@@ -539,6 +544,10 @@ void tasksInit(void)
     setTaskEnabled(TASK_ALTHOLD, true);
```

Comment:
```suggestion
    setTaskEnabled(TASK_ALTHOLD, sensors(SENSOR_BARO | SENSOR_GPS));
```

Reply:
thanks.
And for pos_hold:
`setTaskEnabled(TASK_POSHOLD, SENSOR_GPS);`

Reply:
see above